packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -1147,22 +1147,18 @@ describe("IModelTransformer", () => {
   });
 
   it("remap root subject to root subject", async () => {
-    const sourceIModelName: string = "source";
-    const targetIModelName: string = "target";
-    const sourceIModelFile: string = path.join(
-      outputDir,
-      `${sourceIModelName}.bim`
+    const sourceIModelName: string = "source.bim";
```

Comment:
when you're explicitly setting a variable liek you are here, no need to declare the type.

```suggestion
    const sourceIModelName = "source.bim";
```